docs/burlingame.md

```diff
@@ -0,0 +1,5 @@
+# bgame restaurants 
+- straights
+- urban bistro
+- roti
```

Comment:
(place the link to roti here) http://www.yelp.com/search?find_desc=Roti+Indian+Bistro&find_loc=Burlingame%2C+CA